cmake/Summary.cmake

```diff
@@ -118,6 +118,7 @@ function(caffe_print_configuration_summary)
   caffe_status("  USE_LEVELDB       :   ${USE_LEVELDB}")
   caffe_status("  USE_LMDB          :   ${USE_LMDB}")
   caffe_status("  ALLOW_LMDB_NOLOCK :   ${ALLOW_LMDB_NOLOCK}")
+  caffe_status("  USE_NCCL          :   ${USE_NCCL}")
```

Comment:
Can you add this down below with the other CUDA options?
https://github.com/NVIDIA/caffe/blob/v0.15.1/cmake/Summary.cmake#L144-L148